.github/workflows/go.yml

```diff
@@ -9,11 +9,11 @@ jobs:
         go: [1.16, 1.17]
     steps:
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@331ce1d993939866bb63c32c6cbbfd48fa76fc57 #v2
         with:
           go-version: ${{ matrix.go }}
       - name: Check out source
-        uses: actions/checkout@v2
+        uses: actions/checkout@722adc63f1aa60a57ec37892e133b1d319cae598 #v2
```

Comment:
Where did you obtain this hash?  I see the following:

```
Download action repository 'actions/setup-go@v2' (SHA:331ce1d993939866bb63c32c6cbbfd48fa76fc57)
Download action repository 'actions/checkout@v2' (SHA:5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f)
```

The first hash for `setup-go` matches, but the one for `checkout` doesn't.

Reply:
I got the hash from https://github.com/actions/checkout/commit/722adc63f1aa60a57ec37892e133b1d319cae598
https://github.com/actions/checkout/releases/tag/v2.0.0

Reply:
Ok, thanks.  That explains it.  We've been using 2.3.4 since around November.  Mind using that one instead to avoid downgrading the action?

It should be: 5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f from https://github.com/actions/checkout/releases/tag/v2.3.4

Reply:
If you'll get this updated to the v2.3.4 hash since that is what is already in use, I'll get it merged.

Reply:
I will get it done today. Thanks for the reminder!

Reply:
It is done. Thanks